lsp-headerline.el

```diff
@@ -89,15 +89,14 @@ for caching purposes.")
 
 (defun lsp-headerline--filename-with-icon (file-path)
   "Return the filename from FILE-PATH with the extension related icon."
-  (let ((filename (f-filename file-path))
-        (file-ext (f-ext file-path)))
-    (if file-ext
-        (when (require 'lsp-treemacs nil t)
-          (if-let (icon (lsp-treemacs-get-icon file-ext))
-              (format "%s %s"
-                      (lsp-headerline--fix-image-background icon)
-                      filename)
-            filename))
+  (let ((filename (f-filename file-path)))
+    (if-let* ((file-ext (f-ext file-path))
```

Comment:
can you change to -if-let* since it is breaking the CI for emacs 25.x. Most likely if-let* is not present in emacs 25.

> lsp-headerline.el:90:44:Error: ‘(file-ext (f-ext file-path))’ is a malformed function
make: *** [compile] Error 1

Reply:
done, just noticed the failing CI, works on my setup though (emacs 28.0)